sql/src/main/java/io/crate/analyze/relations/RelationAnalyzer.java

```diff
@@ -709,6 +716,13 @@ public AnalyzedRelation visitTableFunction(TableFunction node, StatementAnalysis
         }
         Function function = (Function) symbol;
         FunctionIdent ident = function.info().ident();
+
+        if (ident.name().equals(UnnestFunction.NAME)) {
```

Comment:
Could you elaborate a bit on why this is necessary? I thought https://github.com/crate/crate/pull/9465/files#diff-bf12c5e61f9b0fe950aa518b1ed601abR796 would make something like this here unnecessary.

As it is it breaks encapsulation, so if it is really required we should at least add some comments that justify it.

Reply:
dropping it, it is not needed. i cannot really recall what was the reason for adding it...